llvm/include/llvm/Frontend/OpenMP/ClauseT.h

```diff
@@ -5,8 +5,41 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-#ifndef FORTRAN_LOWER_OPENMP_CLAUSET_H
-#define FORTRAN_LOWER_OPENMP_CLAUSET_H
+// This file contains template classes that represent OpenMP clauses, as
+// described in the OpenMP API specification v5.2.
```

Comment:
Nit: I think it's better not to explicitly mention the spec version here and just specify it for each clause, as it's done below. This way we'll avoid inconsistencies when inevitably something from a later spec is added to this file and definitions become a mix of standard versions.

```suggestion
// described in the OpenMP API specification.
```